src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1971,4 +1971,19 @@
   <data name="UseSpanClearInsteadOfFillTitle" xml:space="preserve">
     <value>Prefer 'Clear' over 'Fill'</value>
   </data>
+  <data name="PreventNumericIntPtrUIntPtrBehavioralChangesDescription" xml:space="preserve">
+    <value>Built in '+', '-' and explicit casting operators now may or may not throw when overflowing, while it was before. Wrap the expression with 'checked' or 'unchecked' statements to restore old behavior.</value>
```

Comment:
I think it might be clearer to split into functions that used to throw and now don't vs functions that didn't throw and now can.

Having a single error that describes two completely opposing behaviors is confusing, IMO.

Reply:
It is kind of a main description for the analyzer, overall what it does, there is 3 separate error messages for each scenario

Reply:
How is this sound now: `Some built in operators added in .NET 7 behave differently than the old user defined operators. Some operators that used to throw in unchecked context while overflowing will not throw anymore unless wrapped within checked context, and some operators that not used to throw in checked context now would unless wrapped within unchecked context.`

Reply:
> Some built in operators added in .NET 7 behave differently

Is there a place that lists the exact set and what changed? I think this sounds fine for a high-level description, but devs are going to need to know exactly what changed.